ui/fisher/fisher-ui.js

```diff
@@ -75,7 +75,7 @@ export default class FisherUI {
       bar.centertext = row.getAttribute('data-fish');
 
       // Step one: fill until the minimum time
-      if ((min && min !== 'undefined') && (max && max !== 'undefined')) {
+      if (min && min !== 'undefined' && (max && max !== 'undefined')) {
```

Comment:
```suggestion
      if (min && min !== 'undefined' && max && max !== 'undefined') {
```

I love that it only caught one of the parens here??